Pages/fashionCategoryPage.js

```diff
@@ -0,0 +1,31 @@
+class FashionCategoryPage{
+    constructor(){
+        this.title ="Fashion Category"
+    }
+    get brownJacket(){
```

Comment:
Is it possible to have only one getter and one method for clicking on items,
for example get Item and clickOnItem, and in parameters send which item needs to be clicked.

Reply:
I added these kinds of getters and setters, so that I remove code duplication